src/viz/interactivity/viewport-features/ViewportFeaturesGenerator.ts

```diff
@@ -146,51 +215,88 @@ export class ViewportFeaturesGenerator {
     return isInside;
   }
 
-  private getSelectedTiles(): ViewportTile[] {
-    if (!this.deckLayer || !this.deckLayer.state) {
-      return [];
+  private getFeatureId(feature: GeoJSON.Feature): string {
+    if (!feature.properties) {
```

Comment:
We cannot get a feature without properties but with feature id?

Reply:
That's a good point. I had assumed it "contained" in the properties, but I guess we should also consider the feature.id option